packages/stream_chat/lib/stream_chat.dart

```diff
@@ -25,6 +25,7 @@ export 'src/core/api/responses.dart';
 export 'src/core/api/stream_chat_api.dart';
 export 'src/core/error/error.dart';
 export 'src/core/http/interceptor/logging_interceptor.dart';
+export 'src/core/http/stream_http_client.dart';
```

Comment:
❔ was this needed?

Reply:
I noticed that we allow to override `chatApi` in `StreamChatClient`, however, to build `StreamChatApi` it's required to have access to `StreamHttpClientOptions` which is part of `stream_http_client.dart`.

Reply:
makes sense